frontend/src/Components/inputs/TextInput.tsx

```diff
@@ -33,7 +33,18 @@ export function TextInput({
                 value: length,
                 message: `${label} should be ${length} characters or less`
             }
-        })
+        }),
```

Comment:
Can you remove the 'username' field from this? Only the first and last name fields are allowed spaces.


Also can you remove all the changes to the backend from this PR? I included a more robust fix for this issue in #480. I should have referenced the issue